package.json

```diff
@@ -42,12 +42,13 @@
     "eslint-plugin-standard": "^4.0.0",
     "file-loader": "^1.1.6",
     "flow-bin": "^0.93.0",
-    "flow-typed": "^2.5.1",
+    "flow-typed": "=2.5.1",
```

Comment:
This is to fix the `Error: Malformed npm package name!` error, look at [here](https://github.com/flow-typed/flow-typed/issues/3363)